### PR TITLE
Update from-dicom.ipynb

### PR DESCRIPTION
**Pull request by guanfada**
I corrected "z" to "y" in figure labels.
Other changes will be made later.

### CODE REVIEW

**Review by SimonBiggs — APPROVED** [review]
Beautiful :slightly_smiling_face:. A really nice "test the process" PR. Once the tests pass I'm happy to merge this through (one of the tests runs the notebooks and makes sure that they work as expected).

**Comment by SimonBiggs** [comment]
Heads up, this appears to have broken the docs tests:

https://github.com/pymedphys/pymedphys/runs/4452987084?check_suite_focus=true

You can build the docs locally and check that they work by running:

```
poetry run pymedphys dev docs
```

File: lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb
Comment (SimonBiggs) [inline lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:19]:
```suggestion
    "import matplotlib.pyplot as plt",
```

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:19]: I suspect this is the reason the docs broke...

Comment (guanfada) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:19]: that's right. How can I edit the from-dicom.ipynb in my local computer? using the Jupyter-lab? 
image

Comment (SimonBiggs) [reply lib/pymedphys/docs/lib/howto/gamma/from-dicom.ipynb:19]: Yeah, jupyterlab should do the trick :slightly_smiling_face:

**Comment by SimonBiggs** [comment]
Also, for the next PR I've given you commit access:

https://github.com/pymedphys/pymedphys/invitations

That way you can directly make branches within the PyMedPhys repo itself.

**Comment by guanfada** [comment]
I edited the from-dicom.ipynb locally and have performed: 
`poetry run pymedphys dev docs`
so, what is the next step for me to update it in github so that you can review the new effects of the changes? 
I viewed it locally and it looks ok :-)

**Comment by SimonBiggs** [comment]
Make sure that the following has been run:

```
poetry run pre-commit install
```

That above command is only required once on your machine within your current git directory.

Then run:

```
git add -A
git status  # Look at this status, make sure it looks okay
git commit -m "a useful message goes here"
# This will likely fail at this point due to pre-commit installed above

git add -A
git status  # Look at this status, make sure it looks okay
git commit -m "a useful message goes here"
# it should now pass

git push
```

**Comment by guanfada** [comment]
All done and Passed.
What is the next?

**Comment by SimonBiggs** [comment]
> All done and Passed. What is the next?

Did you do `git push`?

**Comment by SimonBiggs** [comment]
Ahh I see what's happened. You need to go:

```
git checkout patch-1
git merge main
git push
```

This is because quite likely when you were editing this notebook locally you were on your local main branch, not the branch corresponding to this PR (patch-1)

**Comment by guanfada** [comment]
I got this error

> 
image

>

**Comment by SimonBiggs** [comment]
Hmm, what does the running the following give you?

```
git remote -v
```

**Comment by guanfada** [comment]
image

**Comment by SimonBiggs** [comment]
And what does the following say:

```
git status
```

**Comment by SimonBiggs** [comment]
It looks like you should probably run:

```
git checkout -b a-new-branch-name-relevant-to-your-pr
git push
```

**Comment by guanfada** [comment]
image

**Comment by SimonBiggs** [comment]
After running:

```
git checkout -b a-new-branch-name-relevant-to-your-pr
git push
```

You can then go to:

https://github.com/pymedphys/pymedphys/compare

To create a new pull request based on your newly created branch

**Comment by guanfada** [comment]
it asked me to run:
 git push --set-upstream origin a-new-branch-name-relevant-to-your-pr

image

**Comment by SimonBiggs** [comment]
Yup, that's right. Yeah do that step :)

**Comment by SimonBiggs** [comment]
The new PR is over at https://github.com/pymedphys/pymedphys/pull/1591

**Pull request closed without merging**
